shared/actions/teams/index.js

```diff
@@ -42,17 +42,30 @@ const _leaveTeam = function(action: Constants.LeaveTeam) {
   })
 }
 
+function getPendingConvParticpants(state: TypedState, conversationIDKey: ChatConstants.ConversationIDKey) {
+  if (!ChatConstants.isPendingConversationIDKey(conversationIDKey)) return null
+
+  return state.chat.pendingConversations.get(conversationIDKey)
+}
+
 const _createNewTeamFromConversation = function*(
   action: Constants.CreateNewTeamFromConversation
 ): SagaGenerator<any, any> {
   const {payload: {conversationIDKey, name}} = action
   const me = yield select(usernameSelector)
   const inbox = yield select(selectedInboxSelector, conversationIDKey)
+  var participants
+
   if (inbox) {
+    participants = inbox.get('participants').toArray()
```

Comment:
this branch sets participants to a JS array, but the else branch it's still an Immutable array. I think you want to remove the `.toArray()` here and move it down to the for loop, like `for (const username of participants.toArray())`